packages/eslint-plugin/src/configs/eslint-recommended.ts

```diff
@@ -23,7 +23,7 @@ export = {
         'no-this-before-super': 'off', // ts(2376)
```

Comment:
```suggestion
        'no-this-before-super': 'off', // ts(2376) & ts(17009)
```
https://github.com/eslint/eslint/pull/17500#discussion_r1311669537
[playground](https://www.typescriptlang.org/play?#code/MYGwhgzhAECCbQN4ChkF9WkjAQgI2gFMAPAF0IDsATGeJZaR6AMwHtWAuaCUgJwEsKAcwZNgrCj14BXYKVa8AFAEp6TddFIALfhAB0bVtAC80AORgzojd2kAHQkuXXoGDMixRoAYWBEylDRwCCjq4pJ8svJKvIRgVBIgAJ4s7FxSgkKqoTaaOvqGLuoQ9o4qLm5AA)